pkg/lang/ir/v1/r.go

```diff
@@ -31,26 +31,28 @@ func (g generalGraph) installRLang(root llb.State) llb.State {
 }
 
 func (g generalGraph) installRPackages(root llb.State) llb.State {
+
 	if len(g.RPackages) == 0 {
 		return root
 	}
-	// TODO(terrytangyuan): Support different CRAN mirrors
-	var sb strings.Builder
+
 	mirrorURL := "https://cran.rstudio.com"
 	if g.CRANMirrorURL != nil {
 		mirrorURL = *g.CRANMirrorURL
 	}
-	sb.WriteString(fmt.Sprintf(`R -e 'options(repos = c(CRAN = "%s")); install.packages(c(`, mirrorURL))
-	for i, pkg := range g.RPackages {
-		sb.WriteString(fmt.Sprintf(`"%s"`, pkg))
-		if i != len(g.RPackages)-1 {
-			sb.WriteString(", ")
-		}
+
+	lib := "/usr/local/lib/R/site-library/"
+
+	root = root.
+		Run(llb.Shlexf("chmod 711 %s", lib), llb.WithCustomNamef("[internal] setting execute permision for default R package library for envd users")).Root()
```

Comment:
How come only execute permission is required? Can `envd` user install packages?

Reply:
711 doesn't allow users to install packages. Just changed it to 777. `Execute` is needed because user need it to load installed library